src/librustc/Cargo.toml

```diff
@@ -10,14 +10,15 @@ crate-type = ["dylib"]
 
 [dependencies]
 arena = { path = "../libarena" }
+bitflags = "1.0"
 fmt_macros = { path = "../libfmt_macros" }
 graphviz = { path = "../libgraphviz" }
 jobserver = "0.1"
 log = "0.3"
 owning_ref = "0.3.3"
 rustc_back = { path = "../librustc_back" }
-rustc_bitflags = { path = "../librustc_bitflags" }
 rustc_const_math = { path = "../librustc_const_math" }
+rustc_cratesio_shim = { path = "../librustc_cratesio_shim" }
```

Comment:
I think because this depends on libsyntax this also isn't needed

Reply:
(copied from the other comment) True, but that feels fragile to me. What's the downside to always including this?